packages/code-du-travail-frontend-e2e/features/outils/preavis-retraite.feature

```diff
@@ -48,3 +48,47 @@ Fonctionnalité: Outil - Préavis de retraite
 
     Quand j'ouvre l'accordion
     Alors je vois "Durée prévue par le code du travail (durée légale)"
+
+  Scénario: Parcours en connaissant sa convention collective
+    Soit un utilisateur sur la page "/outils/preavis-retraite"
+
+    Alors je vois "Étapes"
+    Alors je vois "Préavis de départ ou de mise à la retraite"
+    Alors je vois "permet de calculer la durée de préavis à respecter en cas de départ ou de mise à la retraite"
+
+    Quand je clique sur "Commencer"
+
+
+    Alors je vois "Qui est à l’origine du départ en retraite"
+    Quand je choisis "Le salarié décide lui-même de partir à la retraite"
+    Quand je clique sur "Suivant"
+
+    Alors je vois "Quel est le nom de la convention collective applicable ?"
+    Quand je clique sur "Suivant"
+    Alors je vois "Vous devez répondre à cette question"
+    Quand je choisis "Je connais ma convention collective"
+    Alors je vois "Précisez et sélectionnez votre convention collective"
+    Quand je renseigne "843" dans le champ "Nom de la convention collective ou son numéro d’identification IDCC"
+    Alors j'attends que le texte "Boulangerie" apparaisse
+    Quand je clique sur "Boulangerie"
```

Comment:
C'est cette instruction qui ne fonctionne pas les tests e2e, aucune idée de pourquoi, le texte est bien présent pourtant...

Reply:
Peut-être le scroll to ou le waiting pose soucis ? <https://github.com/SocialGouv/code-du-travail-numerique/blob/ce176b9b1daf6e1831e5c48555a1f72ed3a78dd1/packages/code-du-travail-frontend-e2e/step_definitions/global.js#L73>

Reply:
J'ai essayé mais le problème est toujours là :( je n'avais pas pensé à regarder à ce niveau. Je vais prendre quelques minutes pour y jeter un oeil 👍

Reply:
https://github.com/SocialGouv/code-du-travail-numerique/pull/4226#issuecomment-1029894836

Reply:
J'ai corrigé le warning `Warning: validateDOMNesting(...): <form> cannot appear as a descendant of <form>.` et j'ai toujours le soucis 😢 C'est nul les test e2e 😛  😭